core/spec/models/spree/order_updater_spec.rb

```diff
@@ -56,13 +56,17 @@ module Spree
       end
 
       it "update order adjustments" do
+        # TODO: Make this into a more valid test, so that stubbing
```

Comment:
I'm actually encouraged that this test is breaking. It's testing an obviously wrong thing (that the adjustment totals aren't updated despite being wrong).